src/NUnitTestAdapter/TestConverter.cs

```diff
@@ -40,61 +40,66 @@ public TestConverter(TestLogger logger, string sourceAssembly)
         /// using the best method available according to the exact
         /// type passed and caching results for efficiency.
         /// </summary>
-        public TestCase ConvertTestCase(ITest test)
+        public TestCase ConvertTestCase(XmlNode testNode)
         {
-            if (test.IsSuite)
+            if (testNode.Name != "test-case")
```

Comment:
I don't know the whole codebase well enough, but should we also check if testNode is null?